tools/autograd/load_derivatives.py

```diff
@@ -561,6 +561,11 @@ def stride_expr(name: str) -> str:
             'nctype': lambda name: NamedCType(name, BaseCType(intArrayRefT)),
             'expr': stride_expr,
         }),
+        # replace self.layout() with self_layout
+        (r'{}.layout\(\)', {
+            'suffix': '_layout',
+            'nctype': lambda name: NamedCType(name, BaseCType(layoutT)),
+        }),
```

Comment:
This change is required so that Tensor is not saved on the backward pass, only its layout.

Reply:
Perfect!